contrib/connector/fc/fibreChannel.go

```diff
@@ -81,6 +81,14 @@ func connectVolume(connMap map[string]interface{}) (map[string]string, error) {
 		return nil, err
 	}
 
+	dmPath, err := getMultipathDevice(deviceWWN)
```

Comment:
Please help me to understand the multipath use case here..
Which API of OpenSDS depends on this functionality ? Volume Attach? 
 Why do we need to collect information's from the host to attach? Do we support attachment only to the host where dock is running?

Reply:
yes, volume attach involves this multipath function. if there are multiple paths connecting between host and storage, we must use multipath device instead of normal sd device to utilize the multipath HA.

Reply:
multipath device is a host dev file, like /dev/dm-*, we need to find out this file path and use it.

Reply:
we can attach volume to any host that opensds aware, not only for dock host.

Reply:
ok